hack/kustomize-sub.sh

```diff
@@ -18,8 +18,9 @@ set -o nounset
 set -o pipefail
 
 root=$(dirname "${BASH_SOURCE[0]}")/..
-kustomize="${root}/hack/tools/bin/kustomize"
-envsubst="${root}/hack/tools/bin/envsubst"
-make --directory="${root}" "${kustomize##*/}" "${envsubst##*/}"
+# shellcheck source=hack/common-vars.sh
+source "${REPO_ROOT}/hack/common-vars.sh"
```

Comment:
- `REPO_ROOT` isn't defined in this file.
- `root` at line 20 should be updated to `REPO_ROOT`

Reply:
this is resolved but here git is not showing